apps/main/lib/main/repositories/articles_repo.rb

```diff
@@ -1,8 +1,11 @@
 require "blog/repository"
+require "main/entities/article"
 
 module Main
   module Repositories
     class ArticlesRepo < Blog::Repository[:articles]
+      struct_namespace Main::Entities
+
```

Comment:
wasn't too clear what these 2 changes meant to config? (esp requiring article but declaring namespace for entities)